python/pyspark/sql/tests/connect/test_parity_udtf.py

```diff
@@ -76,7 +76,6 @@ def test_udtf_with_analyze_using_file(self):
     def test_udtf_access_spark_session(self):
         super().test_udtf_access_spark_session()
 
-    @unittest.skip("TODO(SPARK-50393): support df.asTable() in Spark Connect")
     def test_df_asTable(self):
         super().test_df_asTable()
```

Comment:
We can remove this now?

Reply:
Also `test_df_asTable_chaining_methods`?

Reply:
Removed thanks!